### PR TITLE
Removing asserts

### PR DESCRIPTION
**Pull request by alankao64**
Continuing #110.

### CODE REVIEW

**Comment by 95-martin-orion** [comment]
Interesting - this appears to be a real, globally-visible branch, as opposed to the temp branches seen in tests. Any idea what changed?

**Comment by alankao64** [comment]
I thought that in the tests we were using accounts that were not part of the qFlex developing team, where as this account is part of the team so allowed to push branches.

File: src/tensor.cpp [hunk 1]
```diff
@@ -765,17 +764,17 @@ size_t Tensor::num_zeros() const {
   return count;
 }
 
-void Tensor::print() const {
-  std::string print_str("Tensor of rank ");
-  print_str += std::to_string(_indices.size());
-  print_str += ": ";
+std::string Tensor::tensor_to_string() const {
```
Comment (95-martin-orion) [inline src/tensor.cpp]: Since we also have `print_data`, calling this something like `summary_string` might help distinguish between the two.

**Comment by alankao64** [comment]
ran into merge issues, new branch assert_removal will take care of this PR.

**Comment by s-mandra** [comment]
See #204.

**Pull request closed without merging**